config-examples/peer-a-config.toml

```diff
@@ -0,0 +1,18 @@
+public_key = "peer-a-public-key"
+secret_key = "peer-a-secret-key"
+listen = ["[::]:10001"]
```

Comment:
Do we already support multiple sockets?

Reply:
Kind of. There is a couple of caveats. Currently, we are always responding from the first socket, which will error on ipv4/ipv6 problems, but that is already a TODO

Reply:
So you would consider this preparatory work for multiple udp sockets?   Would it be hard to just add that as part of this PR?